packages/react/test/specs/components/Dropdown/Dropdown-test.tsx

```diff
@@ -137,7 +138,7 @@ describe('Dropdown', () => {
       )
     })
 
-    it('calls onOpen with highlightedIndex set to defaultHighlightedIndex', () => {
+    it('is defaultHighlightedIndex value at first opening', () => {
```

Comment:
nit-nit: just to be on par with the naming used fo rother test cases, lets add `prop` word, to better describe origin of `defaultHighlightedIndex`:

> is defaultHighlightedIndex **prop** value at first opening